### DISCUSSION TITLE
Mortal的工作原理是什么？怎么感觉这里的打法不太对啊

### DISCUSSION THREAD

**Discussion by chucklu** [Equim-chan/mjai-reviewer#126] [Q&A]
https://mjai.ekyu.moe/report/1bd0a2b05f72e7aa.html#kyoku-6-0
雀魂牌谱: https://game.maj-soul.com/1/?paipu=231006-01e66c16-b953-41fc-944d-2181952206ea_a217782092
这里的南4局0本场，第3巡自摸南，
ai为啥建议打4p?34p不是一个两面的搭子嘛
![image](https://github.com/Equim-chan/mjai-reviewer/assets/3004057/da8b93ca-ac7d-4e3c-8c9c-b86e0dc4e448)

![image](https://github.com/Equim-chan/mjai-reviewer/assets/3004057/358eacac-0928-4f09-a0cd-240df5714a28)

**Comment by Equim-chan** [comment]
我认为这里应该是因为 AL 大 TOP 然后配牌很差，就选择了配牌弃和的路线。

另外，issue 是仅用于 bug 报告的，其他问题请使用 discussion。其次，这个问题也不属于这个仓库的范畴，你可以试着先去别的地方问问看(例如社区群组、论坛等)。

**Reply by chucklu** [reply]
了解了，谢谢。另外发现这里https://mjai.ekyu.moe/report/609a7fd5cf1c04eb.html#kyoku-4-0 和牌居然被Mortal跳过
难道是因为只有1000点？
![image](https://github.com/Equim-chan/mjai-reviewer/assets/3004057/e0214cad-979a-4ca5-a057-1f33c28bdbfe)
